server/grpc_service.go

```diff
@@ -133,6 +177,15 @@ func (s *Server) Tso(stream pdpb.PD_TsoServer) error {
 
 // Bootstrap implements gRPC PDServer.
 func (s *Server) Bootstrap(ctx context.Context, request *pdpb.BootstrapRequest) (*pdpb.BootstrapResponse, error) {
+	receiverAddr := getReceiverAddr(ctx)
```

Comment:
Maybe we can create an issue about using gRPC Interceptor like [`grpc-proxy`](https://github.com/mwitkow/grpc-proxy) in the later.